clang/lib/Basic/Sanitizers.cpp

```diff
@@ -43,6 +44,23 @@ std::optional<double> SanitizerMaskCutoffs::operator[](unsigned Kind) const {
 
 void SanitizerMaskCutoffs::clear(SanitizerMask K) { set(K, 0); }
 
+std::optional<std::vector<int>>
+SanitizerMaskCutoffs::getAllScaled(int ScalingFactor) const {
+  std::vector<int> scaledCutoffs;
+
+  bool anyNonZero = false;
+  for (unsigned int i = 0; i < SanitizerKind::SO_Count; ++i) {
+    int scaled = round((operator[](i)).value_or(0) * ScalingFactor);
```

Comment:
it should be lround

and better to clamp in doubles before lround

Reply:
Changed to lround

See earlier comment about clamping